apiserver/facades/agent/uniter/package_test.go

```diff
@@ -12,3 +12,5 @@ import (
 func TestPackage(t *stdtesting.T) {
 	coretesting.MgoTestPackage(t)
 }
+
+//go:generate mockgen -package mocks -destination mocks/newlxdprofile.go github.com/juju/juju/apiserver/facades/agent/uniter LXDProfileBackendV2,LXDProfileMachineV2,LXDProfileUnitV2,LXDProfileCharmV2
```

Comment:
Should be `//go:generate go run github.com/golang/mock/mockgen ...` so that we use go mod.

Reply:
fixed here... and a few others which were missing it as a driveway